src/planscape/collaboration/models.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.gis.db import models
+from django.contrib.contenttypes.models import ContentType
+from django.utils.translation import gettext_lazy as _
+from django.contrib.auth import get_user_model
+import uuid
+
+User = get_user_model()
+
+
+class Role(models.TextChoices):
```

Comment:
I'm not sure if it's better to do it this way or to use Postgres enums, but I suppose if we ever move to (or want to support) a different database, this gives us that flexibliity. :+1:

Reply:
I remember that you mentioned using enums so I went with this, but maybe we should use postgres enums? Do you have an example on how to do that?

Reply:
On second thought, it doesn't look like Django migrations will create the enum type in postgres without the help of another library, like [django-pg-fields](https://django-pgfields.readthedocs.io/en/latest/search.html?q=enum&check_keywords=yes&area=default), and this adds a lot more complexity to our model.

I think what you did here is a good solution, and it seems to be the recommendation from the docs now, anyway.